website/docs/security/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
+  "label": "Security",
```

Comment:
how can we hide categories under security?

Reply:
So I gave it a go, and it's being annoying. It looks like the only way to do that is to describe the whole navigation by hand.

Reply:
moved as page, now it looks better